indico/legacy/webinterface/pages/conferences.py

```diff
@@ -314,32 +315,24 @@ def _getTitle(self):
         return entry.localized_title
 
 
-class WPConferenceModifBase(main.WPMainBase):
-    def __init__(self, rh, conference, **kwargs):
-        conference = getattr(conference, 'as_legacy', conference)
-        main.WPMainBase.__init__(self, rh, **kwargs)
-        self._navigationTarget = self._conf = conference
+class WPConferenceModifBase(WPEventManagement):
+    """Base class for event management pages without Jinja inheritance.
 
-    def getJSFiles(self):
-        return (main.WPMainBase.getJSFiles(self) +
-                self._includeJSPackage('Management') +
-                self._asset_env['modules_event_cloning_js'].urls() +
-                self._asset_env['modules_event_management_js'].urls())
+    Do not use this for anything new.  Instead, use `WPEventManagement`
```

Comment:
Let's make sure we mention this in the weekly meeting (and **emphasize**!) (delete the comment)